symfony/client/src/orangehrmAdminPlugin/pages/education/EditEducation.vue

```diff
@@ -0,0 +1,149 @@
+<!--
```

Comment:
Since these UIs belong to qualifications section, I highly suggest you to move this from `education` to `qualificationEducation` as we have followed that convention everywhere else. check #733

```
src/orangehrmAdminPlugin/pages/qualificationEducation/EditQualificationEducation.vue
src/orangehrmAdminPlugin/pages/qualificationEducation/SaveQualificationEducation.vue
src/orangehrmAdminPlugin/pages/qualificationEducation/QualificationEducation.vue 
```